content/en/docs/reference/command-line-tools-reference/feature-gates/strict-cos-enforcement-for-VAP.md

```diff
@@ -0,0 +1,15 @@
+---
+title: StrictCostEnforcementForVAP
+content_type: feature_gate
+_build:
+  list: never
+  render: false
+
+stages:
+  - stage: alpha
+    defaultValue: false
+    fromVersion: "1.30"
```

Comment:
If these features were introduced in v1.30, we might want to target the _main_ branch for this PR, which currently has the latest v1.30 docs?

Reply:
Makes sense. I'll close this one and create another PR against the main branch then. Thanks!

Reply:
```suggestion
    fromVersion: "1.31"
```
?